wrap/swig/sharedPointers.i

```diff
@@ -7,7 +7,7 @@
 
 %import SiconosConfig.h
 
-#if (SICONOS_CXXVERSION >= 201103L) && !defined(USE_BOOST_FOR_CXX11)
+#if defined(SICONOS_STD_TO_STRING) && !defined(SICONOS_USE_BOOST_FOR_CXX11)
 #define STD11 std
```

Comment:
I think you meant SICONOS_STD_SHARED_PTR: the following bits of code are from g++-v4/bits/shared_ptr.h